test/integration/lib/bulk_download_service_test.rb

```diff
@@ -121,6 +121,45 @@ class BulkDownloadServiceTest < ActiveSupport::TestCase
     end
   end
 
+  test 'should support Windows-formatted paths in output path map' do
+    study_file = @study.metadata_file
+    directory = @study.directory_listings.first
+    path_map = BulkDownloadService.generate_output_path_map([study_file], [directory], os: 'Windows')
+    path_map.values.each do |output_path|
+      assert output_path.match(%r{\\}).present?,
+             "#{output_path} did not match Windows-formatted paths: #{output_path}"
+      assert_not output_path.match(%r{/}).present?,
+                 "#{output_path} matched Mac OSX formatted path when it should not: #{output_path}"
```

Comment:
Not using the new styling ("macOS") is fine, but if using the old styling then let's use its [canonical form](https://en.wikipedia.org/wiki/MacOS).  It helps when naively searching, e.g. via grep.

```suggestion
                 "#{output_path} matched Mac OS X formatted path when it should not: #{output_path}"
```